src/main/java/seedu/duke/data/timetable/Event.java

```diff
@@ -83,4 +88,8 @@ public Boolean getRecurring() {
     public void setRecurring(Boolean recurring) {
```

Comment:
I used the wrong "Boolean" when creating the class, can change it to "boolean"

Reply:
Noted, will change. But it won't cause an error as JRE handles boxing and unboxing of the non-primitive types in the background.